gradle/libs.versions.toml

```diff
@@ -97,6 +97,11 @@ mail = "1.4.1"
 rome = "1.0"
 jettison = "1.1"
 thrift = "0.12.0"
+derby = "10.14.2.0"
+datanucleus-core = "4.1.17"
+datanucleus-api-jdo = "4.2.4"
+datanucleus-rdbms = "4.1.19"
+datanucleus-jdo = "3.2.0-m3"
```

Comment:
Are they just used for test? If not, you should also update the license.bin.

Reply:
I saw they are already in.
https://github.com/apache/gravitino/blob/4f961bb2d498fb6158b4f1db0b249e8a1b356149/LICENSE.bin#L323

https://github.com/apache/gravitino/blob/4f961bb2d498fb6158b4f1db0b249e8a1b356149/LICENSE.bin#L279